apps/web/src/app/(public)/login/LoginForm/index.tsx

```diff
@@ -40,9 +44,27 @@ export default function LoginForm({
           placeholder='Ex.: jon@example.com'
           errors={errors?.email}
         />
-        <Button fullWidth isLoading={isPending && !error}>
-          Login
-        </Button>
+        <div className='flex flex-col gap-6'>
+          <Button fullWidth isLoading={isPending && !error}>
+            Login
+          </Button>
+
+          <div className='relative'>
+            <Separator />
+            <div className='absolute left-1/2 top-1/2 -translate-x-1/2 -translate-y-1/2'>
+              <div className='bg-background px-2'>
+                <Text.H6 color='foregroundMuted'>Or</Text.H6>
+              </div>
+            </div>
+          </div>
+
+          <Button variant='outline' fullWidth asChild>
```

Comment:
You have `iconProps` no need to use `<Icon />` inside buttons

Reply:
nah because it's full width it does not do it the way i want it